.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Console Logs**
+If applicable, add console logs to help give more information about your problem.
+
+**System Information (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Version (If not applicable skip): [e.g Ubuntu 22.04 LTS/Windows Server 2022]
```

Comment:
I think you should move the example for Version to OS and change Version to Commit or smth, since I assume this is the system details of the system running fosscord rather than the client?

Reply:
oh ye that part is actually missing the part about commit hash and all that. its something i need to add.